templates/assets/css/vlabs-style.css

```diff
@@ -367,4 +366,8 @@ iframe {
 
 .simulation-header .dropdown .vlabs-hidden {
   display: none;
+}
+
+.f-1{
+  flex:1;
```

Comment:
this class name is very unintuitive. There's already a bootstrap class flex-grow-1 for the same purpose, please use that

Reply:
I tried with flex-grow, but it was not giving desired result. Hence I updated the class name to expand-1 for better readability.